src/marin/processing/classification/consolidate.py

```diff
@@ -151,11 +200,17 @@ def apply_filter_classify(input_data: dict, doc_filter: FilterConfig, id_to_attr
         value = attribute_value
 
     # Check both lower and upper bounds if specified
-    if doc_filter.threshold is not None and value < doc_filter.threshold:
-        return False
+    if doc_filter.reverse:
```

Comment:
This is kind of hard to follow. maybe do something like:

accepted = True

if doc_filter.label is not None:
   ...
  accepted = False

if doc_filter.threshold is not None and value < threshold:
  accepted = accepted & accepted

...

if reverse:
  return not accepted
return accepted